cmd/operator-sdk/generate/csv_prompt_util.go

```diff
@@ -0,0 +1,114 @@
+// Copyright 2020 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package generate
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strings"
+
+	log "github.com/sirupsen/logrus"
+)
+
+// getInputDisplayName prompts the user to provide the displayname that
+// describes the operator, trims spaces and returns the input provided
+// by the user.
+func getOperatorDisplayName() string {
+	fmt.Printf("Displayname for the operator: ")
+
+	userDisplayName := readString(bufio.NewReader(os.Stdin))
+	if userDisplayName == "" {
+		fmt.Printf("Empty string cannnot be accepted")
+		return getOperatorDisplayName()
+	}
+	return userDisplayName
+}
+
+// getProviderName prompts the user to provide the name of the provider
+// of the operator, trims spaces and returns the input provided
+// by the user.
+func getProviderName() string {
+	fmt.Printf("The name of the publishing entity behind the operator: ")
+	providerName := readString(bufio.NewReader(os.Stdin))
+	if providerName == "" {
+		fmt.Printf("Empty string cannnot be accepted")
+		return getProviderName()
+	}
+	return providerName
+}
+
+// getProviderURL prompts the user to provide the URL of the provider and
+// returns the input provided by the user.
+func getProviderURL() string {
+	fmt.Printf("Any relevant URL for the provider name (if exists): ")
+	providerURL := readString(bufio.NewReader(os.Stdin))
+	return providerURL
+}
+
+// getOperatorKeywords prompts the user to provide the list of keywords describing the
+// operator.
+func getOperatorKeywords() []string {
+	fmt.Printf("List of keywords for your operator in the format :- word1, word2, word3 ... : ")
+	userKeywords := readArray(bufio.NewReader(os.Stdin))
+
+	if len(userKeywords) == 0 || len(userKeywords[0]) == 0 {
+		fmt.Printf("No keywords provided. Please provide keywords in the format " +
+			"word1, word2, word3 ...")
+		return getOperatorKeywords()
+	}
+	return userKeywords
+}
```

Comment:
I would follow a similar pattern and make this `getStringArray(msg string) []string`